templates/android/library/src/main/java/io/appwrite/Client.kt.twig

```diff
@@ -327,8 +327,17 @@ class Client @JvmOverloads constructor(
                         .charStream()
                         .buffered()
                         .use(BufferedReader::readText)
+                        
                     val error = if (response.headers["content-type"]?.contains("application/json") == true) {
-                        body.fromJson()
+                        val map = gson.fromJson<Map<String, Any>>(
```

Comment:
I did not touch this file, this is probably difference between master and feat-perps branch. Syncing feat-prefs branch should solve this.

Reply:
I synced the feat-preps branch, however still see these changes

Reply:
I manually reverted the changes